lesson_02/job1/dal/sales_api.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+from typing import List, Dict, Any
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_URL = 'https://fake-api-vycpfa6oca-uc.a.run.app/sales'
+
+
+def get_sales(date: str) -> List[Dict[str, Any]]:
+    """
+    Get data from sales API for specified date.
+
+    :param date: date to retrieve the data from
+    :return: list of records
+    """
+    token = os.getenv('AUTH_TOKEN')  # Отримуємо токен доступу з environment змінної
+    if not token:
+        raise EnvironmentError("AUTH_TOKEN environment variable is not set in .env.")
+
+    headers = {"Authorization": f"Bearer {token}"}
```

Comment:
```suggestion
    headers = {"Authorization": token"}
```
This "token" is already encoded, so there's no need to specify `"Bearer"` keyword